core/server/services/posts/posts-service.js

```diff
@@ -28,6 +28,12 @@ class PostsService {
                     message: messages.invalidNewsletterId
                 });
             }
+        } else {
+            // Set the newsletter_id if it isn't passed to the API
+            const newsletters = await this.models.Newsletter.findPage({status: 'active', limit: 1, columns: ['id']}, {transacting: frame.options.transacting});
```

Comment:
If this is meant to be finding the default newsletter then I think we'll need an `order: 'sort_order DESC'` param so we get the first newsletter in the sorted list.

Reply:
Or are newsletters always sorted by `sort_order`?

Reply:
They are! Ignore this 🙂